src/adagrams.js

```diff
@@ -1,15 +1,133 @@
-export const drawLetters = () => {
+
+//import { random } from "core-js/core/number";
+
+
+
+export const drawLetters = () =>{
   // Implement this method for wave 1
-};
+  let LETTER_POOL={
+    'A': 9,
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+  };
+  let remainingletterspool=[];
+  //create a varibale (let perhaps)parse out the letters
+  let letterChoice=Object.assign({},LETTER_POOL)
+  while (remainingletterspool.length < 10){
+    let randoNum=(Math.floor(Math.random() * 26))
+    //chooses a random number between 0-26
+    let selectKey=Object.keys(letterChoice)[randoNum]
+    //accesses the key and value associated 
+    if (letterChoice[selectKey]!==0){
+      remainingletterspool.push(selectKey);
+      // adds the key to the empty list
+      letterChoice[selectKey]-=1;
+    } 
+    }// if letter exists pops the key
+    return remainingletterspool;
+    
+  
+
+}
```

Comment:
Overall I see what you were trying to achieve here, just a few small adjustments!

 1) Your function currently forces a false randomness. It will randomly pick one of the 26 letters and remove one, 
 but it ignores the fact that we have different letters with different frequencies. To get around this, the main 
 suggestion would be to make a global dictionary like the one you had, and then write a helper function that 
 converts that dictionary into a list or string that holds every single letter possible. Then we can use that helper 
 function to create our collection of letters as opposed to making a deep copy!

 2) If we have that collection of all the letters, it makes it really easy to remove letters as you use them!